### DISCUSSION TITLE
x86 with issues when using USB flashdrive

### DISCUSSION THREAD

**Discussion by dcliff9** [SmoothWAN/SmoothWAN#109] [General & Help]
I...LOVE...SmoothWAN. Other than I can't get it to work. Probably because I'm a noob.
Setup was beyond easy. I popped the image on a thumb drive and used it to boot this Intel n3700/8gb box I have that has 4 gb ports. Generally on this box, I have Ubuntu installed running the Speedify app and everything works perfectly, but this web interface and appliance-like setup is EXACTLY what I need. So I'm super hopeful!

BUT... I am having this problem where Speedify seems to just kinda freeze every few seconds. The Speedify graph pauses and completely drops a space or two. Then it will try to pick back up for a bit and then freeze again. Sometimes if drops the eth connections entirely and disconnects from Speedify. If I try doing a stream (2500kbps RTMP for example) the stream won't stay going for more than a few seconds until Speedify pauses again. 

The Speedify graph kinda looks like this : .......  ..... .. ...... ..... ..... . ........   ....... .... .... .. .......

If I disconnect Speedify and try streaming directly (without going through Speedify/bonding) everything works smoothly.

I'm hard wired to a comcast connection (that has plenty of bandwidth). Also wired to a T-Mobile 5G Gateway and even had my phone tethered through USB (in case it was an ethernet port issue). Same results. Tried multiple Speedify servers. Tried on each port. Tried different transport modes in Speedify, etc. Tried with header compression on and off, encryption on and off. Made sure to remove the extra WAN and WAN6 interfaces per the instructions. Tried turning on/off packet steering. SQM is off. CPU is ~90% idle while streaming and I don't see any change when these freezes happen. 

Any other ideas that I can try? I reeeeeealy want to get this to work. :)

**Comment by dcliff9** [comment]
In case anyone else runs into a similar problem, it was the thumb drive I was running this off of. I know, I know. NOOB. Have a great weekend!

**Comment by TalalMash** [comment]
Thanks for the feedback, added USB flash drives as not recommended in the Wiki. 
SD cards with readers are recommended if there is no internal storage; slow but they don't block I/O like modern flashdrives (~happens due to the constant error correction routine in high density flash~).
Edit: Some flash drives prepare a pool of erased pages everytime there is a write causing unresponsiveness.
